webdriver/state/text/get_text.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import uuid
+
+from support.asserts import assert_error, assert_success
+from support.inline import inline
+
+def test_getting_text_of_a_non_existant_element_is_an_error(session):
+   session.url = inline("""<body>Hello world</body>""")
+   id = uuid.uuid4()
+
+   result = session.transport.send(
+       "GET",
+       "session/%s/element/%s/text" % (session.session_id, id))
```

Comment:
This is fine, but would like to see a PR to add this endpoint to the webdriver client in the repo as well, so that it isn't necessary for all PRs to use the endpoint directly.